mace/calculators/mace.py

```diff
@@ -234,23 +333,37 @@ def calculate(self, atoms=None, properties=None, system_changes=all_changes):
         )
         for i, model in enumerate(self.models):
             batch = batch_base.clone()
-            if self.model_type in ["AtomicChargesMACE", "EnergyChargesMACE"]:
-                out = model(
-                    batch.to_dict(),
-                    compute_stress=compute_stress,
-                )
-            else:
-                out = model(batch.to_dict(), compute_stress=compute_stress)
+            model_out = model(
```

Comment:
here you are first computing energy and charges with`model.forward` and then the derivatives (both forces and of the charge cv) with `self._get_outputs`, right?

i find it a little bit confusing that some of the outputs come from `model_out` and some from `out`. for instance the energy is passed through `self._get_outputs` while the charges and the node_energies are not. maybe it is better to pass everything that has been computed from forward to self._get_outputs and then access them from the same dict?